worker/buildbot_worker/test/unit/test_scripts_upgrade_slave.py

```diff
@@ -35,13 +35,13 @@
 """
 
 
-class TestUpgradeSlave(misc.IsBuildslaveDirMixin,
-                       misc.FileIOMixin,
-                       misc.LoggingMixin,
-                       unittest.TestCase):
+class TestUpgradeWorker(misc.IsWorkerDirMixin,
```

Comment:
This test will work :) However whole upgrade step will be removed, since `buildbot_worker` will not have any versions (yet) to upgrade from. I'll do it in separate PR.